bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/HDPowerViewWebTargets.java

```diff
@@ -163,9 +164,37 @@ public HDPowerViewWebTargets(HttpClient httpClient, String ipAddress) {
      * @throws HubProcessingException if there is any processing error
```

Comment:
Ok, minor point but I would add @returns to the Java Doc

Reply:
> Ok, minor point but I would add @returns to the Java Doc

Whoops, that slipped - thanks! Fixed.